spec/api.yaml

```diff
@@ -108,6 +111,19 @@ paths:
                     type: string
                 example:
                   { "message": "Invalid Login details" }
+        400: 
```

Comment:
Make this an error 401 Unauthorized instead. Why are you returning the OTP? It is something you send via email, not in the response.